spec/factories/follower.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/pivotal-tracker/follower'
+
+FactoryGirl.define do
+  factory :follower, class: PivotalTracker::Follower do
+    sequence(:id) { |n| n }
+    name          { Faker::Name.name }
+    email         { Faker::Internet.email }
+    initials      { name.split.map{|name| name[0] } }
+    username      { Faker::Internet.user_name }
+    kind          'kind'
```

Comment:
Ex. for this one would be `kind: follower`?

![follower](https://cloud.githubusercontent.com/assets/18703294/22600742/19391c3e-e9f9-11e6-9de3-0caf194cf648.jpeg)